spec/services/bulk_upload/sales/year2024/row_parser_spec.rb

```diff
@@ -282,6 +282,26 @@ def purch_id_field
           expect(questions.map(&:id)).to eql([])
         end
       end
+
+      describe "#validate_nulls" do
+        context "when non-setup questions are null" do
+          let(:attributes) { setup_section_params.merge({ field_32: "" }) }
+
+          it "fetches the question's check_answer_label if it exists, otherwise it gets the question's header" do
```

Comment:
This reads like it would be better as two different test cases. Is there a reason that doesn't work?